dev/benchmarks/c/ipc_benchmark.cc

```diff
@@ -0,0 +1,178 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <stdio.h>
+
+#include <benchmark/benchmark.h>
+
+#include <nanoarrow/nanoarrow.hpp>
+#include <nanoarrow/nanoarrow_ipc.hpp>
+
+static ArrowErrorCode MakeFixtureInputStreamFile(const std::string& fixture_name,
+                                                 ArrowIpcInputStream* out) {
+  const char* fixture_dir = std::getenv("NANOARROW_BENCHMARK_FIXTURE_DIR");
+  if (fixture_dir == NULL) {
+    fixture_dir = "fixtures";
+  }
+
+  std::string fixture_path = std::string(fixture_dir) + std::string("/") + fixture_name;
+  FILE* fixture_file = fopen(fixture_path.c_str(), "rb");
+
+  NANOARROW_RETURN_NOT_OK(ArrowIpcInputStreamInitFile(out, fixture_file, true));
```

Comment:
Please add option name
```suggestion
  NANOARROW_RETURN_NOT_OK(ArrowIpcInputStreamInitFile(out, fixture_file, /*xxx=*/ true));
```